### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by elusyo**
* Update the Makefile with a make check command. It launches all the tests in the tests/ directory through the test.sh script.
* It is possible to add a test in tests/ by creating a new directory with a test.sh within (permissions a+x) where you write your tests. Attention, the script is executed by another script : all the relative paths you use are from the tests/ directory.
* The test script calculate successes and failures
*  Add a first simple test. It executes distar with 2 arguments and wait for an "Ok" answer

### CODE REVIEW

File: tests/good_output/test.sh [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-#           Copyright (C) 2018            #
-#   Yann Régis-Gianas - Étienne Marais    #
-###########################################
-
-
-##  Test if the simple output is good
+set -euC
+cd "$(dirname "$0")"
 
 # Program output
```
Comment (yurug) [inline tests/good_output/test.sh]: Are you sure that `distar` exists at this point? Imagine I have just cloned the repository and run `make check`, is `distar` built before the testsuite is run?

**Comment by yurug** [comment]
Please add a travis file.

File: tests/good_output/test.sh (hunk 1 above)
Comment (elusyo) [reply tests/good_output/test.sh]: Yes it is build by the jbuilder command : 
```jbuilder
(deps ((file ../src/distar.exe)
```

Comment (yurug) [reply tests/good_output/test.sh]: Great!

**Comment by elusyo** [comment]
Travis Ci works

**Pull request merged**